ethvm/main.cpp

```diff
@@ -150,88 +126,54 @@ int main(int argc, char** argv)
 
 	Ethash::init();
 	NoProof::init();
-
-	for (int i = 1; i < argc; ++i)
-	{
-		string arg = argv[i];
-		if (arg == "-h" || arg == "--help")
-			help();
-		else if (arg == "-V" || arg == "--version")
-			version();
-		else if (arg == "--vm" && i + 1 < argc)
-		{
-			string vmKindStr = argv[++i];
-			if (vmKindStr == "interpreter")
-				vmKind = VMKind::Interpreter;
+	po::options_description transactionOptions("Transaction options");
+	transactionOptions.add_options()
+			("value", po::value<u256>(), "<n>  Transaction should transfer the <n> wei (default: 0).")
+			("gas", po::value<u256>(), "<n>    Transaction should be given <n> gas (default: block gas limit).")
+			("gas-price", po::value<u256>(), "<n>  Transaction's gas price' should be <n> (default: 0).")
+			("sender", po::value<string>(), "<a>  Transaction sender should be <a> (default: 0000...0069).")
+			("origin", po::value<string>(), "<a>  Transaction origin should be <a> (default: 0000...0069).")
+			("input", po::value<string>(), "<d>   Transaction code should be <d>")
+			("code", po::value<string>(), "<d>    Contract code <d>. Makes transaction a call to this contract")
+			("gas-limit", po::value<u256>(), "");
```

Comment:
Couldn't we call maxBlockGasLimit() right here to generate description?

Reply:
Anyway I did this, seems to work fine, please say if you had some reasons to do it your way